build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension
+import org.jetbrains.kotlin.gradle.targets.js.npm.tasks.KotlinNpmInstallTask
+
+plugins {
+    alias(libs.plugins.kotlinx.binaryCompatibilityValidator)
+}
+
+val deploy by tasks.creating() {
+    dependsOn(getTasksByName("publish", true))
+    dependsOn(getTasksByName("publishNpm", true))
+}
+
+// Right now it is used for switching nodejs version which is supports generated wasm bytecode
```

Comment:
```suggestion
// Right now it is used for switching nodejs version which is supports generated wasm bytecode - Remove after updating to Kotlin 2.0
```

Reply:
Actually, I don't think this section will do anything. KGP isn't applied to the root project, so `NodeJsRootExtension` won't be present until later... perhaps it can be removed?

Reply:
Locally it succeeded. I pushed the commit with removing to check on CI.

Reply:
Hmm, on CI [we failed ](https://teamcity.jetbrains.com/buildConfiguration/KotlinTools_KotlinxAtomicfu_BuildAggregatedLinuxWindows/4665709?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true), thus I will not remove it for now

Reply:
Hmm strange! Let's just update the comment and tidy up the K/JS config in a later PR.